### PR TITLE
add home work 11

### PR DESCRIPTION
**Pull request by NataliiaFed**
add home work #11

### CODE REVIEW

File: homework_10,11.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# 1. Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+from datetime import datetime
+
+now = datetime.now()
+current_time = now.strftime("%H:%M:%S")
+
+def my_decorator(func):
+    def deco_func(*args):
```
Comment (klymenok) [inline homework_10,11.py]: Use **kwargs here as well

File: homework_10,11.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+        now = datetime.now()
+        current_time = now.strftime("%H:%M:%S")
+        print(f"Function name: {func}")
+        print(f"Function call time: {current_time}")
+        func(*args)
```
Comment (klymenok) [inline homework_10,11.py]: Use `return` statement here to be able to decorate functions that return some value. And add **kwargs here

**Review by klymenok — CHANGES_REQUESTED** [review]

File: homework_10,11.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        return round(result, 0)
+    return deco_func
+
+@my_decorator
+def multiply(a, b):
+    return a * b
+
+print(multiply(3.55, 7.63))
+
+# # ===============================================================================================
```
Comment (klymenok) [inline homework_10,11.py]: Why is all this code commented off?

File: homework_10,11.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+current_time = now.strftime("%H:%M:%S")
+
+def my_decorator(func):
+    def deco_func(*args, **kwargs):
+        now = datetime.now()
+        current_time = now.strftime("%H:%M:%S")
+        print(f"Function name: {func}")
+        print(f"Function call time: {current_time}")
+        result = func(*args, **kwargs)
+        return round(result, 0)
```
Comment (klymenok) [inline homework_10,11.py]: This code will raise an exception if you decorate a function that returns a string, for example.
Leave it just as `return result`

**Review by klymenok — CHANGES_REQUESTED** [review]

File: homework_10,11.py (hunk 3 above)
Comment (NataliiaFed) [reply homework_10,11.py]: oops, forgot to uncomment

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**